lunes_cms/cms/models/feedback.py

```diff
@@ -0,0 +1,34 @@
+# from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.utils.translation import ugettext_lazy as _
+from django.db import models
+
+
+class Feedback(models.Model):
+    """
+    Contains a title, a description and a reference to the matter of feedback
+    """
+
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    discipline = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    document = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    training_set = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    # options = models.GenericForeignKey("general", discipline, document, training_set)
+    # matter = models.CharField(max_length=12, choices=options, default=options[0])
+
```

Comment:
```suggestion

    def content_object_link(self):
        """
        Include a link to the edit form of the content object in list display

        :return: Link to the edit form of the content object
        :rtype: str
        """
        # Get link to the admin form if the content object
        admin_edit_link = reverse(
            f"admin:cms_{self.content_type.model}_change", args=[self.object_id]
        )
        # Escape label because string is marked as safe afterwards
        escaped_label = escape(str(self.content_object))
        # Mark string as safe to allow html link tag
        return mark_safe(f"<a href={admin_edit_link}>{escaped_label}</a>")

    content_object_link.short_description = _("refers to")

    def content_object_type(self):
        """
        Better readable representation of the content type

        :return: Verbose name of the content object's model
        :rtype: str
        """
        return self.content_object._meta.verbose_name

    content_object_type.short_description = _("content type")
    content_object_type.admin_order_field = "content_type"

```
We could add two read only fields to improve the appearance of the generic foreign key:
- `content_object_link` provides a link to the edit form of the referenced object, so it can directly be improved from the feedback list/entry
- `content_object_type` provides a better representation of the object type